### ISSUE TITLE
Breaking release in the last hour

### ISSUE THREAD

**Issue by TylerBarnes** [eventualbuddha/lines-and-columns#207]
It looks like there may have been either a breaking change or bug introduced for `v1` starting at `1.2.0` in the last hour.
Was that intentional and is a fix being worked on?

Looks like it breaks Gatsby sites which are using this package via `postcss-loader`

```
16:34:37 PM:
ERROR Generating JavaScript bundles failed

16:34:37 PM:
require() of /usr/src/app/www/node_modules/lines-and-columns/build/index.js from /usr/src/app/www/node_modules/parse-json/index.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.

16:34:37 PM:
require() of ES modules is not supported.
```

**Comment by eventualbuddha** [comment]
This was a mistake. It should be fixed in v1.1.10 and v1.2.3, either of which should be pulled in with a dependency on `^1.1.6` or `~1.1.6` as pretty much any dependent package likely has. Can you verify this is no longer an issue?

**Comment by TylerBarnes** [comment]
Thanks 😄 that's good to know 👍 I'm trying it now

**Comment by TylerBarnes** [comment]
I can confirm 1.2.3 fixes it! 🙌 You're a hero @eventualbuddha 💪 thanks for this package

**Comment by eventualbuddha** [comment]
Glad it's fixed for you! And honestly this whole debacle probably cost more time than this package has ever saved for anyone 😢 Oh well.

**Comment by jakub300** [comment]
Hi @eventualbuddha, 1.2.3 adds engines.node in package.json limiting it to node 12+ which wasn't the case in earlier 1.x releases. It would be great if you could remove it for backward compatibility.

**Comment by eventualbuddha** [comment]
@jakub300 good point. Published v1.1.11 and v1.2.4 to address that.
